wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/class-plugin.php

```diff
@@ -297,8 +297,9 @@ public function log_translation_source( GP_Translation $translation ) {
 		$source = '';
 		if ( 'GP_Route_Translation' === GP::$current_route->class_name ) {
 			if ( 'import_translations_post' === GP::$current_route->last_method_called ) {
-				$source                           = 'import';
+				$http_referer                     = stripslashes( $_POST['_wp_http_referer'] );
 				$this->imported_translation_ids[] = $translation->id;
+				$source                           = preg_match( '/^\.\.\/\?sort/', $http_referer ) ? 'import_from_playground' : 'import_from_frontend';
```

Comment:
I now added a `<input type="hidden" name="source" value="translate-live" />` so you can check for `isset( $_POST['source'] && 'translate-live' === $_POST['source']`

Reply:
Can we use the sources `playground` and `frontend`?